README.md

```
@@ -8,86 +8,85 @@
+It aims to complement Git with everything necessary for hardware development to substitute commercial Product Lifecycle
+Management (PLM) tools. It's a free versioning and change management solution for all your CAD needs, built around your
+CAD artifacts instead of being built into your CAD tool.
 
 Browse [our documentation] and visit [our website]. Watch our 💥💥[demos](https://youtube.com/@PartCAD)💥💥.
 
 ## Join us!
 
-Stay informed and share feedback by joining [our Discord server](https://discord.gg/zdwyxkGM).
-<br/>
-Subscribe on [LinkedIn], [YouTube], [TikTok], [Facebook], [Instagram], [Threads] and [Twitter/X].
+Stay informed and share feedback by joining [our Discord server](https://discord.gg/zdwyxkGM). <br/> Subscribe on
```

Markdownlint (0.35.0)

Element: br
Inline HTML

(MD033, no-inline-html)